### PR TITLE
Prepare to rename master to main

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ For example, any given Jaeger installation at Uber is typically processing sever
 ### Native support for OpenTracing
 
 Jaeger backend, Web UI, and instrumentation libraries have been designed from the ground up to support the [OpenTracing standard](https://opentracing.io/specification/).
-  * Represent traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
+  * Represent traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/main/specification.md#references-between-spans)
```
Comment (joe-elliott) [inline README.md:50]: this 404s for me

**Review by joe-elliott — COMMENTED** [review]
Similar question to the other repos. Is there a reason to keep functionality for both master and main? Is that just something we want during a transition period?

File: README.md (hunk 1 above)
Comment (jpkrohling) [reply README.md:50]: Good catch, this shouldn't have been renamed.

**Comment by jpkrohling** [comment]
> Is that just something we want during a transition period?

Once we switch completely to main, I'll send another PR removing master from the CI files.

**Review by joe-elliott — DISMISSED** [review]

**Comment by jpkrohling** [comment]
> There are a lot of references to jaegertracing/jaeger-ui/blob/master/... in the changelog. Should this be updated to use main?

GitHub sets a redirect, we should be fine for the old ones. For the new ones, we should use the new branch name. Although it wouldn't be hard to rename it as part of this PR...

**Comment by jpkrohling** [comment]
Changelog changed.

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**